Dockerfile

```diff
@@ -0,0 +1,125 @@
+ARG BASE_IMAGE=nvcr.io/nvidia/l4t-pytorch:r32.4.3-pth1.6-py3
+FROM ${BASE_IMAGE}
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV JUPYTER_PASSWORD=jetbot
+
+WORKDIR /jetbot_dir
+
+ENV SHELL /bin/bash
+
+RUN apt-get update && apt-get install -y
+
+#
+# install OpenCV (with GStreamer support)
+#
+COPY jetson-ota-public.asc /etc/apt/trusted.gpg.d/jetson-ota-public.asc
+
+RUN echo "deb https://repo.download.nvidia.com/jetson/common r32.4 main" > /etc/apt/sources.list.d/nvidia-l4t-apt-source.list && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+            libopencv-python \
+    && rm /etc/apt/sources.list.d/nvidia-l4t-apt-source.list \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install traitlets (master, to support the unlink() method)
+RUN echo -e "\e[48;5;172m Install traitlets \e[0m"
+#sudo python3 -m pip install git+https://github.com/ipython/traitlets@master
+RUN python3 -m pip install git+https://github.com/ipython/traitlets@dead2b8cdde5913572254cf6dc70b5a6065b86f8
+
+# Install jupyter lab
+RUN echo -e "\e[48;5;172m Install Jupyter Lab \e[0m"
+RUN apt-get update
+RUN apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs libffi-dev 
+RUN pip3 install jupyter jupyterlab
+RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager
+
+RUN jupyter lab --generate-config
+RUN python3 -c "from notebook.auth.security import set_password; set_password('${JUPYTER_PASSWORD}', '/root/.jupyter/jupyter_notebook_config.json')"
+
+# Install jupyter_clickable_image_widget
+RUN echo "\e[42m Install jupyter_clickable_image_widget \e[0m"
+RUN cd && \
+    apt-get install -y libssl1.0-dev && \
+    git clone https://github.com/jaybdub/jupyter_clickable_image_widget && \
+    cd jupyter_clickable_image_widget && \
+    git checkout tags/v0.1 && \
+    pip3 install -e . && \
+    jupyter labextension install js && \
+    jupyter lab build
+
+# install jetbot python module
+RUN cd /opt && \
+    git clone https://github.com/NVIDIA-AI-IOT/jetbot/ && \
+    cd jetbot && \
+    apt install -y python3-smbus && \
+    apt-get install -y cmake && \
+    python3 setup.py install && \
+    cp -r notebooks ${WORKDIR}/
```

Comment:
Is workdir in this instance the mounted volume?  Does this work?

Reply:
no, so I took care of populating this with `docker_setup_workdir.sh`.
We cam remove this line.